### ISSUE TITLE
[feat] Make enableOnVimEnter work when Shatur/neovim-session-manager is used

### ISSUE THREAD

**Issue by fnune** [shortcuts/no-neck-pain.nvim#240]
## Describe the problem

These opts work as intended:

```lua
  {
    "shortcuts/no-neck-pain.nvim",
    version = "*",
    opts = {
      autocmds = {
        enableOnVimEnter = true,
      }
    },
  },
```

However, when I enable [`Shatur/neovim-session-manager`](https://github.com/Shatur/neovim-session-manager), it stops working. It seems like `neovim-session-manager`'s effect keeps working but prevents `enableOnVimEnter` from working.

## Describe the solution

Make both plugins work in tandem somehow.

**Comment by sand4rt** [comment]
somewhat related to: https://github.com/shortcuts/no-neck-pain.nvim/issues/221

**Comment by shortcuts** [comment]
> somewhat related to: #221

Definitely, I might need to think of a FAQ page in https://github.com/shortcuts/no-neck-pain.nvim/wiki with your proposed solution, wdyt?

**Comment by sand4rt** [comment]
Hey @shortcuts, I could add that, but I don't have access to edit the wiki

**Comment by shortcuts** [comment]
It seems like I can't grant access to a single contributor :( it's either everyone can edit it or no one except me

**Comment by shortcuts** [comment]
sessions are supported now https://neovim.io/doc/user/usr_21/#_sessions
